scene/gui/popup_menu.cpp

```diff
@@ -2894,7 +2778,7 @@ bool PopupMenu::get_allow_search() const {
 }
 
 String PopupMenu::get_tooltip(const Point2 &p_pos) const {
-	int over = _get_mouse_over(p_pos);
+	const int over = _get_mouse_over(p_pos);
```

Comment:
```suggestion
	int over = _get_mouse_over(p_pos);
```

We prefer not to use `const` for local variables with primitive types, it's just creating visual noise. Same for other cases.